elementary/utils/log.py

```diff
@@ -36,7 +62,7 @@ def get_console_handler():
 
 
 def get_file_handler():
-    file_handler = logging.FileHandler(LOG_FILE)
+    file_handler = CustomFileHandler(LOG_FILE, delay=True)
```

Comment:
I think we should add the file handler dynamically with the correct log file output, initializing it with `LOG_FILE` and then changing it in `emit` feels a bit hacky...

I think it's better that we'll add a line of the format:
```logger.addHandler(get_file_handler(log_path))```

to `elementary/monitor/cli.py` (where `log_path` is constructed from the target path parameter.